starboard/tools/api_leak_detector/api_leak_detector.py

```diff
@@ -395,9 +401,25 @@ def main():
   def IsSbSymbol(symbol):
     return symbol.startswith('Sb') or symbol.startswith('kSb')
 
+  # Allowed POSIX symbols in Starboard 16
+  allowed_sb16_posix_symbols = [
+      'calloc',
+      'free',
+      'malloc',
+      'posix_memalign',
+      'realloc',
+  ]
+
+  def IsAllowedPosixSybol(symbol, sb_api_version):
+    if int(sb_api_version) == 16:
+      return symbol in allowed_sb16_posix_symbols
+    else:
+      return False
+
   leaked_symbols = set(
       symbol for symbol in ProcessNmOutput(nm_output) \
           if symbol not in allowed_c99_symbols and not IsSbSymbol(symbol)
```

Comment:
This conditional is too long, I think the list comprehension should just be `symbol for symbol in ProcessNmOutput(nm_output) if not IsAllowedSymbol(symbol)` with calls to the other functions in `IsAllowedSymbol`

Reply:
Done